modules/provision-oci-infrastructure-ocp-cluster-temp.adoc

```diff
@@ -0,0 +1,165 @@
+
+.Procedure
+
+. From the navigation menu, select *Developer Services* > *Stacks*.
+
+. Click *Create Stack*. 
+
+.. Select *My configuration* for the origin of the Terraform configuration.
+
+. Under *Stack Configuration*, select the *Zip file* checkbox and click *Browse*.
+
+. Select the zip file from your local drive and click *Next*. 
+
+. Complete the fields for the stack variables: 
++
+[cols="1,3",options="header",subs="quotes"]
+|===
+|Field |Action required
+
+|*Compartment*
+|Select the child compartment you created previously.
+
+|*Cluster Name*
+|Specify the name of your cluster, such as `oci`. Use the same cluster name that you entered during the creation of
+the ISO image in the Red Hat Hybrid Cloud Console. For details, see "Using the Assisted Installer to generate an OCI-compatible discovery ISO image".
+
+|*OpenShift Image Source URI*
+|Enter the Pre-Authenticated Request URL created in the previous task. For details, see "Provisioning OCI infrastructure for your cluster".
+
+|*Control Plane Shape*
+|Instance shape of the control plane nodes. Currently shapes are available for virtual machines and bare metal. For details, see link:docs.oracle.com/en-us/iaas/Content/Compute/References/computeshapes.htm[Compute Shapes].
+
+|*Compute Shape*
+|Instance shape of the compute nodes. Currently shapes are available for virtual machines and bare metal. For details, see link:docs.oracle.com/en-us/iaas/Content/Compute/References/computeshapes.htm[Compute Shapes].
+
+|*Zone DNS*
+|Specify DNS name server that stores DNS records for a zone, for example `openshift-demo.devcluster.openshift.com`. This is the same value as the base domain in {oci}.
+
+|*Enable Private DNS*
+|Set the DNS zone to public or private. Specifying a value of `true`, creates a private DNS zone. Specifying a value of `false` creates a public DNS zone. For a private DNS zone, you must configure your local `/etc/hosts` file to reach the cluster.
+|===
+
+. (Optional) Edit the default values for the other fields as needed. By default, the
+cluster has 3 control plane nodes and 3 compute nodes.
+
+. Click *Next* and review the stack information and variables.
+
+. Click *Create* to create the stack. The console redirects to the stack details page
+for the new stack. 
+
+. On the stack details page, click *Apply* to create an apply job and provision the infrastructure for the cluster.
++
+After running an apply job, get the job's details to check its status. Succeeded (SUCCEEDED) indicates that the job has completed.
+
+== Getting the custom manifests
+
+After provisioning the infrastructure in the Resource Manager service, from the "Outputs" of the stack job, get the “dynamic_custom_manifest”. This output contains all the required manifests, concatenated and preformatted with the configuration values for CCM and CSI. 
+
+The dynamic_custom_manifest file contains the following manifests: *not sure if this is still correct*
```

Comment:
not sure we need to describe the content, it's up to Oracle to decide what they put in it.